app/dashboard/add/page.tsx

```diff
@@ -1,9 +1,109 @@
+        <div className='flex w-full justify-center'>
+          <Card className='h-56 w-11/12 bg-gray-50' />
+        </div>
+        <div className='flex flex-col gap-2 pl-6'>
+          <h1 className='text-3xl font-extrabold'>Itsu</h1>
+          <h4 className='pb-1 text-sm'>60 Horseferry Rd, London, SW1P 2AF</h4>
+        </div>
+        <div className='flex w-full flex-col items-center justify-center gap-7'>
+          <Card className='w-11/12 bg-gray-50'>
+            <CardContent className='flex gap-3 pb-7'>
+              <div className='flex w-4/6 flex-col gap-3 pt-6'>
+                <Label className='text-md flex w-20 items-center justify-between'>
+                  Food
+                  {' '}
+                  <Cookie size={20} />
+                </Label>
+                <Rate
+                  value={food}
+                  onChange={(v) => {
+                    setFood(v);
+                    console.log(food);
+                  }}
+                  allowHalf
+                  style={{ color: 'black' }}
+                />
+                <Label className='text-md flex w-20 items-center justify-between'>
+                  Value
+                  {' '}
+                  <PiggyBank size={20} />
+                </Label>
+                <Rate
+                  value={value}
+                  onChange={(v) => {
+                    setValue(v);
+                    console.log(value);
+                  }}
+                  allowHalf
+                  style={{ color: 'black' }}
+                />
+                <Label className='text-md flex w-20 items-center justify-between'>
+                  Vibe
+                  {' '}
+                  <Flame size={20} className='mb-1' />
+                </Label>
+                <Rate
+                  value={vibe}
+                  onChange={(v) => {
+                    setVibe(v);
+                    console.log(vibe);
+                  }}
+                  allowHalf
+                  style={{ color: 'black' }}
+                />
+              </div>
+              <div className='flex w-3/6 flex-col items-center gap-2 pt-6'>
+                <Card className='flex h-16 w-24 items-center justify-center'>
+                  <h1 className='text-lg font-bold text-black'>
+                    {numberFixer(food)}
+                  </h1>
+                </Card>
+                <Card className='flex h-16 w-24 items-center justify-center'>
+                  <h1 className='text-lg font-bold text-black'>
+                    {numberFixer(value)}
+                  </h1>
+                </Card>
+                <Card className='flex h-16 w-24 items-center justify-center'>
+                  <h1 className='text-lg font-bold text-black'>
+                    {numberFixer(vibe)}
+                  </h1>
+                </Card>
+              </div>
+            </CardContent>
+          </Card>
+          <div className='flex h-16 w-11/12 items-end gap-2'>
```

Comment:
Rather than setting a fixed height on the div and then setting align: flex-end it would be better to use margin top and bottom. 

```suggestion
          <div className='mx-2 my-4 flex w-11/12 gap-2'>
```